common/src/primitives/version.rs

```diff
@@ -61,6 +61,12 @@ impl TryFrom<String> for SemVer {
     }
 }
 
+impl std::fmt::Display for SemVer {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "v{}.{}.{}", self.major, self.minor, self.patch)
```

Comment:
Perhaps it's better to move that v out of here. I don't see the benefit in forcing everyone to format their version with a v prefix.